integration_tests/helpers/s3Helper.ts

```diff
@@ -69,8 +69,8 @@ const deleteDirectoryRecursiveInS3 = async (
   prefix?: string
 ): Promise<DeleteObjectCommandOutput[]> => {
   const result = await getS3ItemsList(bucketName, prefix);
-  if (result.Contents == null) {
-    throw new Error("No files found to delete");
+  if (result.Contents === null || result.Contents === undefined) {
+   return []
```

Comment:
when running single test which uses this function throws an error if the s3 directory already deleted

Reply:
if we have 
```
  if (result.Contents === null || result.Contents === undefined) {
    throw new Error("No files found to delete");
}
```

does that work?

Reply:
No if we ran the same test again if the directory is already deleted in the previous run and the second run it will throw error that no files to delete